### ISSUE TITLE
CSM: Chaos Lord - MoK doesn't unlock mount / artefact weapon

### ISSUE THREAD

**Issue by Admarilo** [BSData/wh40k-7th-edition#3588]
File/Catalogue: Chaos Space Marines: Codex (2012) v2019

BattleScribe version: 2.00.31

Platform: Android

Dropbox: No

Description:
If Chaos Lord is choosen and Mark of Khorne is selected you can't select juggernaut of khorne as a mount and you can't select the "Axe of Blind Fury" as artefact weapon.

**Comment by cartag** [comment]
Long boring explanation: I had to add a second version of MoK to the Chaos Lord to make other things behave properly. Before it was registering cult units as troops whenever anybody had that one particular mark equipped. I had to add new versions of MoN/K/S to Lord and MoT to Sorcerer. In all the updating, the JuggyLord didn't check for the new MoK, but the old. The Axe also was set to only look for the global MoK, not the specialized one on the lord. 

Tldr: I fixed it.
